client/packages/common/src/ui/components/buttons/standard/DialogButton.tsx

```diff
@@ -29,6 +31,7 @@ interface DialogButtonProps {
   autoFocus?: boolean;
   color?: 'primary';
   type?: 'button' | 'submit' | 'reset';
+  customLabel?: string;
```

Comment:
"Custom label" allows us to keep using this component when we want some buttons to say something very case-specific. Makes it easier to keep consistent when using a Custom button label alongside other "common" buttons (Ok, cancel, etc.)

Reply:
Love this!